_posts/2024-05-10-interactive-bioinformatics.md

```diff
@@ -0,0 +1,70 @@
+Learn how to navigate folders and files through the CLI by playing a spooky game.
+
+- [Bashcrawl](https://gitlab.com/slackermedia/bashcrawl)
+Learn CLI by playing a dungeon crawler.
+
+- [PonyLinux](https://github.com/NCGAS/PonyLinux)
+Another CLI game but with Ponys.
+
+- https://sandbox.bio/
+Perform actual bioinformatics tasks such as alignment and variant calling all in the browser. Several tutorials on alignment, variant calling, CLI and even jq. 
+All through the magic of wasm (biowasm). The exploration sandboxes are worth checking out if you want to grok how some algorithms work. 
+Worthy of it's own blog post with the number of tutorials.
+
+- The Great Clade Race - Zombie Island Edition  https://serc.carleton.edu/teachearth/activities/238347.html
+Learn cladistics theory through a fun game zombie game. This is particularly interesting as it's more DnD-like.
+
+## Games for Science
+
+The follow list of games utilise video games to solve biological questions. By playing the game, you're helping science!
+
+- Fold it - https://fold.it/
+Learn how protein folding works, compete against other players and help predict protein folding all at once.
+
+- EyeWire  https://eyewire.org/
+Traverse real brain scans to mark neurons in 3D space. The music is fantastic but be warned, it's not easy!
+
+- [EteRNA](https://eternagame.org/) 
+A fun RNA molecule building game with a nice community. Avaliable on mobile phones as well.
+
+Both of the below projects are from McGill School of Science. Their website and blogs https://dnapuzzles.org details their adventures. 
+
+- Zooniverse https://www.zooniverse.org/projects
+A large collection of citizen science projects that harnse people to find patterns in biological data, identify animals through visual or sound, or mark astroids and space objects.
+
+- Project Discovery/Borderlands Science 
+In-game minigames in EVE Online and Borderlands Science respectively. A site note on EVE Online: If you like tracking things in spreadsheets, checkout EVE Online. 
+
+## Hardcore
+
+- https://rosalind.info
+Rosalind is a fantastic way to get your hands dirty solving real world computational problems relating to molecular biology. From counting number of nucleotides to computing HMM profiles, its super fun and educational. The difficult ramps up fast.
+
+## Not bioinformatics related but worthy of a mention:
+
+- https://seeing-theory.brown.edu/ Learn statistics through visuals
+- https://setosa.io/ev/principal-component-analysis/ Understanding PCA 
+- https://pair-code.github.io/understanding-umap/ the same for UMAP
+- https://algorithm-visualizer.org/ algorthm visualiser
+- https://carpentries.org/ for learning to pair-code
+- regexone.com - learn regex through interactive sessions
+- Swirl (Learn R in R)
+
+
+A final thought from the amazing Helena:
+
+[screenshot2.png]
+
+Wouldn't that be amazing! 
```

Comment:
```suggestion
## Future Work

If you were looking for an MSc project for a student, it would be a wonderful thing to have more free, open-source, interactive games like these that can be embedded in bioinformatics websites everywhere!
```